### DISCUSSION TITLE
Impossible to apply for a student pack

### DISCUSSION THREAD

**Discussion by soomaroo-S** [community/community#56964] [GitHub Education]
### Hi everyone,

I am a comp sci student who has had the github student pack for a while now
The process to renew it this year feels just impossible

The bot they use to check my proof of academic status keeps rejecting me by saying 

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

> Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

My profile has been updated with the requested info. When I upload official docs as I have them online (as do most people in the 21st century, I imagine), the bot wants me to take a photo. When I take a photo, it doesn't pick up the information it wants *that is clearly present* on each supporting document. 

I really don't know what else I can do here

**Comment by YuChang00** [comment]
me too

**Comment by samus-aran** [comment]
Hi @soomaroo-S 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this section will help the most.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance 👍
